### PR TITLE
Fix strict mode storybook

### PR DESCRIPTION
**Pull request by LFDanLu**
Use this branch for strict mode testing + fixes for now. Will explore fixing this via the strict mode toggle PR

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

## 🧢 Your Project:

RSP

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -24,9 +24,9 @@ export const parameters = {
 };
 
 export const decorators = [
-  Story => (
+  story => (
```
Comment (snowystinger) [inline .storybook/preview.js:27]: you know, we can probably get rid of this decorator, there are story options that do this now using layout: center
https://storybook.js.org/docs/react/configure/story-layout

Comment (LFDanLu) [reply .storybook/preview.js:27]: Will investigate this option later for the decorator I'll be adding to support scrolling in all stories

**Review by snowystinger — APPROVED** [review]

File: .storybook/preview.js [hunk 2]
```diff
@@ -24,9 +24,9 @@ export const parameters = {
...
     <VerticalCenter style={{alignItems: 'center', minHeight: '100vh', boxSizing: 'border-box', display: 'flex', justifyContent: 'center'}}>
-      <Story />
+      {story()}
```
Comment (snowystinger) [inline .storybook/preview.js:29]: didn't we make this change originally because it was crashing storybook as you have it now?

**Comment by LFDanLu** [comment]
I confused myself and Rob haha, this PR is no longer needed, superceded by https://github.com/adobe/react-spectrum/pull/3333 now that I've got that one working

**Pull request closed without merging**